database/backend/security.json

```diff
@@ -1 +1,9 @@
-[]
+[
+  {
+    "name": "Django Forbid",
+    "description": "Django Forbid provides documentation for a growing OSS tool forbidding geolocated individuals in forbidden countries, regions, and territories.",
```

Comment:
```suggestion
    "description": "Secure your Django app by controlling the access - grant or deny user access based on device and location, including VPN detection.",
```

Reply:
Hey @Bhanuteja01, thanks for adding [Django Forbid](https://github.com/pysnippet/django-forbid). Could you please change the description of the project as it is in its GitHub repository?